src/lib/gssapi/mechglue/g_init_sec_context.c

```diff
@@ -121,6 +121,7 @@ OM_uint32 *		time_rec;
     gss_OID		selected_mech;
     gss_mechanism	mech;
     gss_cred_id_t	input_cred_handle;
+    OM_uint32		used_req_flags;
```

Comment:
I think this is necessary:

Consider the case when the mechglue has been updated to include `set_context_flags()`, but the particular mech used hasn't; in this case, an application could conceivably use `set_context_flags()` to set the req_flags it wants, and specifies 0 to `init_sec_context()`, expecting the flags from `set_context_flags()` to be used. However, if the mech doesn't support `set_context_flags()`, it has no way of learning of these flags, unless we detect that the argument `req_flags` to `init_sec_context()` is 0 and substitute in those from the `set_context_flags()` call (Line 213).